app/models/concerns/orchestration/dhcp.rb

```diff
@@ -19,6 +20,20 @@ def dhcp_record
     @dhcp_record ||= (provision? && jumpstart?) ? Net::DHCP::SparcRecord.new(dhcp_attrs) : Net::DHCP::Record.new(dhcp_attrs)
   end
 
+  def rebuild_dhcp
+    begin
+      if dhcp?
+        set_dhcp
+      else
+        logger.info "DHCP not supported for #{host.name}, #{ip}, skipping orchestration rebuild"
```

Comment:
Nitpick, use #{self} rather than host.name here, as DHCP might be on a non-primary interface, so showing the NIC (self) it's on will be more precise.